extensions/api/streaming_api.py

```diff
@@ -89,9 +89,15 @@ async def _handle_chat_stream_message(websocket, message):
             "speaker": generate_params['silero_tts_speaker'],
             "voice_pitch": generate_params['silero_tts_voice_pitch'],
             "voice_speed": generate_params['silero_tts_voice_speed'],
-            "eleven_labs_speaker": generate_params['eleven_labs_speaker'],
         })
     
+    if generate_params['elevenlabs_enable'] == True:
```

Comment:
As previously mentioned here https://github.com/MRK-TDY/text-generation-webui/pull/2#discussion_r1499713955, we want to have a three-state behavior when enabling TTS: no tts, silero only, elevenlabs then silero.

So, only one `if` statement is needed, where you check whether silero or elevenlabs are enabled.


Another possibility is to change the `silero_tts_enable` to be an enum with the following possibilities: `disabled`, `silero`, `elevenlabs`. In this case, we'd drop the `elevenlabs_enable` variable.